src/Guard.php

```diff
@@ -258,15 +258,15 @@ public function getTokenValue(): ?string
      */
     public function getTokenNameKey(): string
     {
-        return $this->prefix . '_name';
+        return $this->prefix . '-name';
```

Comment:
I understand why, but I'm no sure if it is a breaking change

Reply:
was thinking about it too, but should not be for the framework itself, only if someone have hardcoded reference checks or something similar. 
As i did first -.-  haha.
As tokens to expire and on failure it generates a new one, so in theory it should work still with update mid-production.

Reply:
I just checked README file. We do not talk about hardcoded value, only getting these values and names from public methods `Guard::getTokenNameKey()` and `Guard::getTokenValueKey()`.
It should not be a breaking change, if users are not using hardcoded values.